public/theme/boltcms-2020/featureslist.twig

```diff
@@ -30,7 +30,7 @@
             {% if feature.image is not empty %}
                 <div class="col-md p-4 zoom-images" >
                 <a href="{{ feature.image|thumbnail(1600, 1600) }}"/>
-                    <img class="img-fluid" src="{{ feature.image|thumbnail(600, 400) }}" loading="lazy" />
+                    <img class="img-fluid" src="{{ feature.image|thumbnail(600, 400) }}" loading="lazy" alt="feature">
```

Comment:
Can we use the image alt attribute form Bolt?

Which will be:

`{{ feature.image.alt }}`

Reply:
So it can look like:

`alt="{{ feature.image.alt }}"`